cdc/sink/sink.go

```diff
@@ -79,7 +65,7 @@ func NewSink(ctx context.Context, sinkURIStr string, filter *filter.Filter, conf
 	case "blackhole":
 		return newBlackHoleSink(opts), nil
 	case "mysql", "tidb":
```

Comment:
```suggestion
	case "mysql", "tidb","mysql+ssl", "tidb+ssl":
```
see: https://github.com/pingcap/ticdc/pull/734/files#diff-d6e2759c88c65cddfd8d333dae3e5b94R85